gatsby/on-route-update.js

```diff
@@ -1,10 +1,15 @@
 const onRouteUpdate = ({ location, prevLocation }) => {
   if (
-    window.newrelic &&
+    window.Tessen &&
     location.hash &&
     location.pathname !== prevLocation?.pathname
   ) {
-    window.newrelic.addPageAction('hash_request', { hash: location.hash });
+    window.Tessen.track({
```

Comment:
Could the `useTessen` hook be used here instead?

Reply:
No it was giving out about that when I tried to use it, was saying it wasn't a react component or some nonsense